### PR TITLE
Add and Redact SampleSensitive to caseUpdated Events

### PR DESCRIPTION
**Pull request by LukeLoze1**
# Motivation and Context
Emit redacted sensitive data on case update

# What has changed
Adds sensitive data and.... redacts it

# How to test?
Run the tests, run the ATs on the ticket

# Links
https://trello.com/c/3ZKcJtYK/3012-sensitive-data-to-be-included-on-case-update-event-but-redacted-5

### CODE REVIEW

File: src/main/java/uk/gov/ons/ssdc/caseprocessor/service/CaseService.java [hunk 1]
```diff
@@ -58,12 +59,20 @@ private EventDTO prepareCaseEvent(Case caze, EventHeaderDTO eventHeader) {
...
+
     caseUpdate.setInvalid(caze.isInvalid());
     if (caze.getRefusalReceived() != null) {
       caseUpdate.setRefusalReceived(RefusalTypeDTO.valueOf(caze.getRefusalReceived().name()));
     } else {
       caseUpdate.setRefusalReceived(null);
     }
+    caseUpdate.setSampleSensitive(caze.getSampleSensitive());
+
+    // Due to time taken only run this is there's sampleSensitive data on the case
```
Comment (NickGrantONS) [inline src/main/java/uk/gov/ons/ssdc/caseprocessor/service/CaseService.java:71]: Time taken? Likely to be sub millisecond. More like that it's a _CPU expensive_ operation, not a particularly time consuming one.

**Review by NickGrantONS — CHANGES_REQUESTED** [review]
PR title is incorrect and comment is misleading, but otherwise good

**Review by NickGrantONS — APPROVED** [review]
ATs passing. Good work 👍

**Review by AdamHawtin — APPROVED** [review]
Looks good, passing ATs locally 👍

**Pull request merged**